src/legacy/trulens_eval/__init__.py

```diff
@@ -0,0 +1,134 @@
+# ruff: noqa: E402
+"""
```

Comment:
Is this idea that this is an update to `trulens_eval` routing imports to `trulens`?